examples/features/opa/README.md

```diff
@@ -23,16 +23,6 @@ NAMESPACE=($(kubectl create -f ../namespace.yaml)[0])
 NAMESPACE=${NAMESPACE:10}
 ```
 
-2. Register namespace in `spire` server:
```

Comment:
Please correct step numbers for the next steps.

Reply:
ok, done